Formula/b/binutils.rb

```diff
@@ -63,6 +63,11 @@ def install
     end
   end
 
+  def post_install
+    # Allow ld to find brew glibc when installed on systems that need it
+    (prefix/"etc").install_symlink etc/"ld.so.conf" if (etc/"ld.so.conf").exist?
```

Comment:
May need to figure out if any `/usr/bin/ld + brew glibc` scenarios. Perhaps if update-alternatives was used to pull in newer GCC as our GCC has binutils dependency which I think gets used.

If we need to deal with `/usr/bin/ld` then may have to be done via shim instead.